package.json

```diff
@@ -49,6 +49,7 @@
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
     "babel-plugin-styled-components": "^1.12.0",
     "commitlint": "^12.0.1",
+    "copyfiles": "^2.4.1",
```

Comment:
This is used in the `tsc` script inside the icons package given that `tsc` only bundles TS code into JS, but we also need to bundle the SVG files so we can access them once we install this package. Will verify this is actually working once is published.